core/templates/pages/exploration-editor-page/services/exploration-states.service.ts

```diff
@@ -306,6 +308,12 @@ export class ExplorationStatesService {
       recordedVoiceovers.voiceoversMapping[contentId] = {};
       this.saveRecordedVoiceovers(stateName, recordedVoiceovers);
     }
+    for (let language in this.entityTranslationsService
+      .languageCodeToEntityTranslations) {
+      this.entityTranslationsService.languageCodeToEntityTranslations[
+        language
+      ].removeTranslation(contentId);
```

Comment:
Prefer `forEach` here.

Reply:
This can be ignored per https://github.com/oppia/oppia/pull/20439#discussion_r1641783600

Reply:
Done / turned to utility function